invenio_rdm_records/cli.py

```diff
@@ -115,31 +119,35 @@ def create_fake_record():
             "publication_date": fake_edtf_level_0(),
             "subjects": [{
                 "subject": fake.word(),
-                "identifier": "subj-1",
-                "scheme": "no-scheme"
+                "identifier": "03yrm5c26",
+                "scheme": "ror"
             }, {
                 "subject": fake.word(),
-                "identifier": "subj-1",
-                "scheme": "no-scheme"
+                "identifier": "03yrm5c26",
+                "scheme": "ror"
             }],
             "contributors": [{
-                "family_name": fake.last_name(),
-                "given_name": fake.first_name(),
-                "type": "personal",
+                "person_or_org": {
+                    "family_name": fake.last_name(),
+                    "given_name": fake.first_name(),
+                    "type": "personal",
+                },
                 "affiliations": [{
                     "name": fake.company(),
-                    "identifiers": {
-                        "ror": "03yrm5c26"
-                    }
+                    "identifiers": [{
+                        "scheme": "ror",
+                        "identifier": "03yrm5c26",
+                    }]
                 }],
                 "role": "rightsholder"
             } for i in range(3)],
-            "dates": [{
-                # No end date to avoid computations based on start
-                "date": fake.date(pattern='%Y-%m-%d'),
-                "description": "Random test date",
-                "type": "other"
-            }],
+            # FIXME: add it back when dumper issue is fixed
```

Comment:
This is a temporary fix for having a working demo setup until the issue is properly fixed in https://github.com/inveniosoftware/invenio-drafts-resources/pull/88/files#diff-839975cf50e15f8d6cbae751b63f629f75adbf5b9c1c6721d62b352d0ea6400eR23

Reply:
looks like Lars just merged that one.

Reply:
Indeed.....but I didn't fix it there :) So, for a local setup still you need to omit the `dates`....